creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -23,7 +23,7 @@ const {
 const {
   getSyncStatus,
   setSyncStatus,
-  checkSyncOverride
+  verifySPOverride
```

Comment:
this function isn't exported by syncUtil

Reply:
whoops, idk why replace-all didn't catch - fixed